.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Issues
```

Comment:
Another 💭 , sometimes folks attach screenshots to things, i.e. https://github.com/CasperSleep/Casper/pull/3051. Where do you see images fitting into this template? Think it is worth to have a callout for them?